src/routes/(app)/product/[id]/+page.svelte

```diff
@@ -103,6 +105,18 @@
 		return true;
 	}
 	const { t, locale } = useI18n();
+	const schema: WithContext<SchemaOrgProduct> = {
+		'@context': `https://schema.org`,
+		'@type': 'Product',
+		name: data.product.name,
+		image: `${$page.url.origin}/picture/raw/${data.pictures[0]._id}/format/${data.pictures[0].storage.formats[0].width}`,
```

Comment:
can you get the first image with >500px width & height? (or the first image if not available)